DogeNet.DAL/DBContext.cs

```diff
@@ -46,20 +48,26 @@ public DBContext(DbContextOptions<DBContext> options)
 
         public virtual DbSet<Status> Statuses { get; set; }
 
-        public virtual DbSet<User> Users { get; set; }
-
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             if (!optionsBuilder.IsConfigured)
             {
-                optionsBuilder.UseSqlServer("Server=DESKTOP-QV2VVS8\\MSSQLSERVER01;Database=DogeNetDB;Trusted_Connection=True;");
+                optionsBuilder.UseSqlServer("Server=LEOBIT-ALIU\\ADRIANSQL;Database=DogeNetDB;Trusted_Connection=True;");
             }
         }
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
+            base.OnModelCreating(modelBuilder);
+
+            modelBuilder.Entity<ApplicationRole>(entity => entity.ToTable(name: "Roles"));
```

Comment:
Is there any other possible way to identify IS entities in context?

Reply:
I've deleted custom mapping from OnModelCreating, there are default tables "AspNet<Entity name>" now

Reply:
![image](https://user-images.githubusercontent.com/77879337/142230045-9b99c426-fd1c-47d6-882f-85f706de2351.png)